app/helpers/application_helper.rb

```diff
@@ -241,11 +241,11 @@ def other_formats_links(&)
   # Returns the theme, controller name, and action as css classes for the
   # HTML body.
   def body_css_classes
-    css = ["theme-" + OpenProject::CustomStyles::Design.identifier.to_s]
+    css = ["theme-#{OpenProject::CustomStyles::Design.identifier}"]
 
-    if params[:controller] && params[:action]
-      css << ("controller-" + params[:controller])
-      css << ("action-" + params[:action])
+    if controller_path && action_name
+      css << ("controller-#{controller_path}")
```

Comment:
Shouldn't slashes be escaped?
```suggestion
      css << ("controller-#{controller_path.tr("/", "-")}"
```

Reply:
While I agree with your sentiment, slashes indeed work and are being used https://github.com/opf/openproject/blob/dev/frontend/src/app/features/bim/ifc_models/pages/viewer/styles/generic.sass#L15-L14 😬 so I'd keep this as is

Reply:
Oh, nice catch!
yep, I read [some info about it](https://mathiasbynens.be/notes/css-escapes) and `/` is a legit character for a css class name. It only needs to be escaped in the css file.

Reply:
Yes, they do work but are ugly as hell and also hard to read compared to other characters as simple dashes. So I am totally in favour of refactoring that 👍